src/components/Result/Result.js

```diff
@@ -25,7 +25,7 @@ class ResultView extends Component {
         const { words, answer } = this.props;
         const otherWords = words.filter((word) => word !== answer);
 
-        if (!answer) {
+        if (typeof answer !== 'string') {
```

Comment:
will `if (answer === null)` work the same?
conditions without negations are easier to understand
(compare with: conditions with negations are not easier to understand)

Reply:
yes, it will 😁